pkg/common/si_helper_test.go

```diff
@@ -1,19 +1,19 @@
 /*
- Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
Those changes are unrelated to this PR. Should we address it in another ticket? I guess there are other files having similar issue.

Reply:
unintended changes because of IDE formatting (I think so). Please check the reason if possible so that we can fix it next time. For now, let it go through for this fix.